src/content/guides/build-performance.md

```diff
@@ -152,6 +155,59 @@ new CommonsChunkPlugin({
 })
 ```
 
+### Avoid Extra Optimization Steps
+
+webpack does extra algorithmic work to optimize the output for size and load performance. These are usually not needed during iterative development:
+
+```js
+{
+  optimization: {
+    removeAvailableModules: false,
+    removeEmptyChunks: false,
+    splitChunks: false,
+  }
+}
+```
+
+### Output Without Path Info
+
+webpack has the ability to generate path info in the output bundle. However, this puts inconsequential garbage collection pressure on projects that bundle thousands of modules. Turn this off in the `options.output.pathinfo` setting:
```

Comment:
What does it mean `inconsequential garbage collection pressure`?
Could you please use simpler wording?